.mailmap

```diff
@@ -18,7 +18,6 @@ Alex Griffing <argriffi@ncsu.edu> alex <argriffi@ncsu.edu>
 Alex Griffing <argriffi@ncsu.edu> argriffing <argriffi@ncsu.edu>
```

Comment:
In fact all entries of the form "A <B> A <B>" and "A <B> <B>" should probably be removed.